avaframe/avaframeCfg.ini

```diff
@@ -4,4 +4,12 @@
 
 [MAIN]
 # Path to avalanche directory
-avalancheDir = data/avaHockeySmoothChannel
+avalancheDir = data/avaHockeySmoothSmall
+
+
+[FLAGS]
+# True if plots shall be plotted to screen
+showPlot = False
+
+# True if comparison to reference shall be performed
+compRef = True
```

Comment:
should this be here? We are the only ones who will compare to the reference no?

Reply:
Ok fine I remove it